.gitpod.dockerfile

```diff
@@ -2,8 +2,13 @@ FROM gitpod/workspace-postgres
 
 # Install Ruby
 ENV RUBY_VERSION=2.7.1
+RUN rm /home/gitpod/.rvmrc && touch /home/gitpod/.rvmrc && echo "rvm_gems_path=/home/gitpod/.rvm" > /home/gitpod/.rvmrc
 RUN bash -lc "rvm install ruby-$RUBY_VERSION && rvm use ruby-$RUBY_VERSION --default"
 
+# Install Node
+ENV NODE_VERSION=12.16.3
```

Comment:
can we read from `.nvmrc` here?

Reply:
Probably could, like I said I just pulled all of this in from @snackattas branch.